### DISCUSSION TITLE
Grey annotations from alert after upgrade from 7 to 9.5

### DISCUSSION THREAD

**Discussion by Selaron** [grafana/grafana#67630] [Alerting]
Hello,

after upgrading from Grafana 7 to 9.5.1 all new annotations from alert are visible in grey color while old alerts are still presented in yellow/red/green on timeseries diagrams.

How can I change the presentation color of the new alert annotations?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
